package.json

```diff
@@ -23,7 +23,7 @@
     "method-override": "2.3.3",
     "moment": "2.10.3",
     "passport": "0.2.2",
-    "passport-saml": "0.5.3",
```

Comment:
Hey @deewhyweb 

I've tested this today:

* ADFS assertion seems to behave differently with `0.15.0` vs `0.5.3`. in `0.5.3`, the signover redirect brings you to an ADFS sign-in webpage where you can auth via a form, but in `0.15.0`, you get an HTTP Basic-Auth prompt instead. To land the version bump, I'd prefer to keep things working in a similar manner.
* If you could address [this](https://github.com/feedhenry-templates/saml-service/pull/13/commits/14bc7a838ac13b547ff4a165c2017cce8db76355#r53574573) small comment, that would be great

To help you debug what's changed, I can setup a single node cluster and give you a place to interact with a test ADFS instance we use? Let me know.